origin.spec

```diff
@@ -417,27 +426,28 @@ if [ "$1" = 0 ]; then
 fi
 
 %files clients
+%license LICENSE
 %{_bindir}/oc
 %{_bindir}/kubectl
 %{_sysconfdir}/bash_completion.d/oc
 
 %if 0%{?make_redistributable}
 %files clients-redistributable
+%dir %{_datadir}/%{name}/linux/
+%dir %{_datadir}/%{name}/macosx/
+%dir %{_datadir}/%{name}/windows/
```

Comment:
I had done this in https://github.com/openshift/origin/pull/6257 as well, I'll pull it out of my PR as it's somewhat tangential to the issue I'm addressing there.